src/features/todos.ts

```diff
@@ -1,9 +1,28 @@
 import { Todo } from '../types/Todo';
 
-export const actions = {};
+type SetAction = { type: 'todos/SET', payload: Todo[] };
+type AddAction = { type: 'todos/ADD', payload: Todo };
+type RemoveAction = { type: 'todos/REMOVE', payload: Todo };
```

Comment:
It's recommended to use the `typeof` keyword when defining action types to ensure consistency and avoid errors. For example, you can declare a constant like `const REMOVE_TODO = 'todos/REMOVE'` and then use `typeof REMOVE_TODO` as the type for the action.